### ISSUE TITLE
New event modal window should have a first focus on the title field

### ISSUE THREAD

**Issue by JuliaKirschenheuter** [nextcloud/calendar#4265]
### Steps to reproduce

1. Click to "New event".
2. Modal window will be opened.
3. See that that the focus is outside the modal window.

### Expected behavior

Focus goes to the first field "Event title"

### Actual behaviour

Focus is outside the modal window

**Comment by JuliaKirschenheuter** [comment]
This is a placeholder Issue for accessibility review. Was fixed until v3.3.2
